doc/source/cluster/kubernetes/getting-started/rayjob-quick-start.ipynb

```diff
@@ -357,17 +357,17 @@
      "output_type": "stream",
      "text": [
       "NAME                                                      READY   STATUS    RESTARTS   AGE\n",
-      "kuberay-operator-6bc45dd644-ckxj4                         1/1     Running   0          3m36s\n",
-      "rayjob-sample-7ps8z                                       1/1     Running   0          2s\n",
-      "rayjob-sample-raycluster-ndrdw-head-tfs89                 1/1     Running   0          3m2s\n",
-      "rayjob-sample-raycluster-ndrdw-small-group-worker-9s8hb   1/1     Running   0          3m2s\n"
+      "kuberay-operator-6bc45dd644-tlsfn                         1/1     Running   0          2m26s\n",
+      "rayjob-sample-raycluster-7965c-head-n6nj8                 1/1     Running   0          117s\n",
+      "rayjob-sample-raycluster-7965c-small-group-worker-nlzwx   1/1     Running   0          117s\n",
+      "rayjob-sample-74pmj                                       1/1     Running   0          2s\n"
      ]
     }
    ],
    "source": [
     "# Step 4.3: List all Pods in the `default` namespace.\n",
     "# The Pod created by the Kubernetes Job will be terminated after the Kubernetes Job finishes.\n",
-    "kubectl get pods --sort-by=.metadata.name"
+    "kubectl get pods --sort-by='.metadata.creationTimestamp'"
```

Comment:
@machichima, can you help me understand where the logic for validating this cell's output is located? Thanks!

Reply:
![image](https://github.com/user-attachments/assets/7d633fe3-1822-4892-9fb2-95480eb8cdb3)

I think it's because the last 5 characters are randomized, so if we sort by name, sometimes `rayjob-sample-xxxxx` comes before `rayjob-sample-raycluster-yyyyy-head-zzzzz`.

Reply:
Oh, make sense.

Reply:
Thank you for the explanation!